submarine-workbench-v2/.gitignore

```diff
@@ -0,0 +1 @@
+coverage/
```

Comment:
Do we need to add `package-lock.json` and `yarn.lock`?

Reply:
I think the lock file is required